front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -4,7 +4,7 @@ import { isNilOrError } from 'utils/helperUtils';
 
 // components
 import Button from 'components/UI/Button';
-import { Icon } from 'cl2-component-library';
+import { Icon, useWindowSize } from 'cl2-component-library';
```

Comment:
I moved useWindowSize in the component library and added the appropriate tests there. Would love to see us moving more of these utilities there in the future.